recipe/meta.yaml

```diff
@@ -42,21 +42,21 @@ requirements:
     - gettext   # [unix]
     - patch     # [unix]
   host:
-    - curl      # [unix]
-    - libcurl   # [unix]
-    - expat     # [unix]
-    - libiconv  # [osx]
-    - openssl   # [unix]
-    - pcre2     # [unix]
+    - curl 7.88.1  # [unix]
+    - libcurl 7.88.1  # [unix]
+    - expat 2.4.9  # [unix]
+    - libiconv 1.16  # [osx]
+    - openssl 3.0.8  # [unix]
```

Comment:
Should this be using the pinning in the cbc?

Reply:
If I'm not mistaken, we decided to be explicit instead of implicit for OpenSSL. @ryanskeith am I right?

Reply:
Alright, we just had a talk and we'll use `{{ openssl }}` and rely on aggregate.